packages/adapter-components/test/filters/referenced_instance_name.test.ts

```diff
@@ -180,9 +196,18 @@ describe('referenced instances', () => {
         ] }),
       }
     )
+    const noIdFieldsParent = new InstanceElement('no_idFieldsParent', noIdFieldsType)
+    const noIdFieldsWithParent = new InstanceElement(
+      'no_idFieldsWithParent',
+      noIdFieldsType,
+      {}, [],
```

Comment:
nit: if using separate lines for args, it should be consistent
```suggestion
      {},
      [],
```

Reply:
If they are empty personally i prefer to put them in the same line since they mean nothing

Reply:
I think the usual convention in these is all-or-nothing...